kolibri/core/assets/src/actions.js

```diff
@@ -0,0 +1,21 @@
+
+const UserKinds = require('./constants').UserKinds;
+
+function logIn(store) {
+  store.dispatch('CORE_SET_SESSION', {
+    kind: UserKinds.ADMIN,
+    facility_id: '1',
+    user_id: '2',
+    username: 'starchy52',
+    fullname: 'Mr. Potato Head',
+  });
```

Comment:
We are going to be treating the session as a model, so we will probably always be giving it an id of 1 as well.